run_test.sh

```diff
@@ -16,6 +16,7 @@ yes 'y' | cookiecutter . --no-input
 
 # Run the tests present inside generate project
 cd hello-world-web;
+fab buildprod
```

Comment:
The current convention is `fab prod build`, so it's runs the `prod` function before `build`. 

e..g `fab prod deploy`, `fab prod configure`.